renderer.js

```diff
@@ -18,6 +18,7 @@ function initCanvas () {
   canvas.width = window.innerWidth * 0.5;
   canvas.addEventListener('click', enterFullscreen);
   document.addEventListener('mozfullscreenchange', exitFullscreen);
+  window.addEventListener('resize', handleResize);  // TODO: Consider a debounce.
```

Comment:
then require debounce like main.js does: https://github.com/nickdesaulniers/joshVR/blob/master/main.js#L6
it's already a dependency: https://github.com/nickdesaulniers/joshVR/blob/master/package.json#L7